Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "qiskit-terra"
-version = "0.20.0"
+version = "0.20.1"
```

Comment:
Good catch on this, I probably would have forgot to bump this. We should try to see if we can come up with a way to keep these in sync. In practice it doesn't matter too much because we don't publish the rust crate anywhere and the version number is never included in anything but the build metadata. But it's good to keep this in sync.